agent/consul/server_test.go

```diff
@@ -254,7 +254,7 @@ func TestServer_JoinWAN_Flood(t *testing.T) {
 		})
 	}
 
-	dir3, s3 := testServer(t)
+	dir3, s3 := testServerDCBootstrap(t, "dc1", false)
```

Comment:
Because of that I would actually like to get rid of the convenience methods and only use `testServerWithConfig` to make that explicit.